Kitodo-DataAccess/src/test/java/org/kitodo/dataaccess/format/xml/XMLReaderTest.java

```diff
@@ -0,0 +1,35 @@
+package org.kitodo.dataaccess.format.xml;
```

Comment:
Missing project license header - please add the license header!

Reply:
Was added.